typespec-extension/package.json

```diff
@@ -42,7 +42,7 @@
     "lib/*.tsp",
     "dist/**",
     "!dist/test/**",
-    "target/emitter.jar"
+    "generator/http-client-generator/target/emitter.jar"
```

Comment:
Is it possible to reference outer folder? e.g. "../core/packages/http-client-generator/target/emitter.jar"?

Reply:
Appears no.

```
  "files": [
    "lib/*.tsp",
    "dist/**",
    "!dist/test/**",
    "../emitter.jar"
  ],
```

And I cannot find the jar in tgz.

Though not an expert on npm pack...